frontend/src/fileReceiver.js

```diff
@@ -62,7 +62,7 @@ class FileReceiver extends EventEmitter {
         });
       })
       .then(([fdata, key]) => {
-        this.emit('decrypting', true);
+        this.emit('decrypting');
```

Comment:
Turns out we don't really need the start/finish part of these events. Removing the finished case simplifies the listeners.